layouts/shortcodes/docs/instrumentation/manual-intro.md

```diff
@@ -0,0 +1,14 @@
+[Manual instrumentation](/docs/concepts/instrumentation/manual/) is the process
+of adding observability code to your application yourself. To do so, you need to
+import OpenTelemetry to your code. If you're developing a library or some other
+component that is intended to be consumed by a runnable binary, then you would
+only take a dependency on the API, because unless the OpenTelemetry SDK, the
```

Comment:
> I, because unless the OpenTelemetry SDK

The sentense feels incomplete.

Reply:
apologies, as said in my other comment it looks like I wrote this being not really working properly. Took @cartermp's proposal!